src/core/containers/ServerHtml.js

```diff
@@ -84,7 +84,7 @@ export default class ServerHtml extends Component {
 
   getFaviconLink() {
     const { _config } = this.props;
-    return `${_config.get('amoCDN')}/favicon.ico`;
+    return `${_config.get('amoCDN')}/favicon.ico?v=2`;
```

Comment:
Could we do this in a more automated fashion? This seems a bit brittle.

Reply:
Do we want a new string to be used for each build, or do we just need to bust the cache this once (for now) because we've got a new icon?

Reply:
Well, updating it only when we change the icon means benefiting from more caching, but we already invalidate the rest of our (much larger) assets on a new build so I don't think caching only the favicon makes much sense. The favicon is tiny and I think browsers already aggressively cache favicons for sites. And this way when it's changed no other configs/tests need changing; it will Just Work™.

Reply:
Good point. It sounds like the right thing to do. I'll give it a shot.

Reply:
I've looked into this and I'm not sure how to get webpack to help us out here. You suggested that our webpack build will append cache-busting strings to URLs, but I don't see any evidence of that, unless you mean the webpack generated assets like javascript, css, images, etc. I don't think we'll be asking webpack to generate the favicon, as it's being served from the cdn, so what we're currently doing with webpack assets doesn't seem like a good fit.

We could do something as simple as using `Date.now()` as a query string, or, if we can get the webpack build id somehow in `ServerHtml.js` we could use that. I'm not sure how to do the latter. What do you suggest, @tofumatt?

Reply:
It's worth noting that our build happens on multiple servers right now, which isn't ideal but means using something like `Date.now()` could result in the same page serving different content. The webpack build is made from a hash of contents IIRC though so it should be consistent; storing it someplace might be nice.

I thought all of our links to things like CSS, JS, etc. were processed by webpack somehow. The asset is served from the CDN like all of our resources, but I think the URLs to those resources are still built by webpack.